src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -164,3 +164,42 @@
     margin-right: 0;
   }
 }
+
+.stem-eligibility-page {
+  max-width: 100%;
+  width: 100%;
+}
+
+
+.stem-eligibility {
+  background-color: #f1f1f1;
+  padding: 0.5em 1em;
+
+  p {
+    margin-bottom: 0.5em;
+  }
+
+  ul {
+    margin-left: 1.5em;
+    margin-top: 0.263em;
+    list-style: none !important;
```

Comment:
This changes the semantics of the HTML and can cause bugs in screenreaders.  Check out [this article talking about a fix](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html) or consider different markup.

Reply:
Switched to `list-style-type` as this is needed for the Review Page otherwise the `list-style-type` applied by `fa-ul` is overridden by other classes applied during Review Page creation